<launch>
  <node pkg="imu_read" type="imu_read" name="imu_read" required="true" output="screen">
    <param name="port" value="/dev/ttyUSB0"/>
    <param name="imu_frame_id" value="imu_base"/>
  </node>
</launch>
